### PR TITLE
Fix duplicated jvm signature

### PR DESCRIPTION
**Pull request by koperagen**
Problem appears when multiple nested DataSchema declarations have property with the same name. Using parent declarations in JvmName argument solves it. I decided to not use package names there

computeNestedName is slightly modified computeFqName from compiler

### CODE REVIEW

File: plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/extensions/IrBodyFiller.kt [hunk 1]
```diff
@@ -218,6 +220,21 @@ private class DataFrameFileLowering(val context: IrPluginContext) : FileLowering
         return declaration
     }
 
+    private fun IrDeclarationWithName.nestedName() = StringBuilder().let { computeNestedName(this, it); it.toString() }
```
Comment (Jolanrensen) [inline plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/extensions/IrBodyFiller.kt:223]: Would `buildString { computeNestedName(this@nestedName, this) }` not be a lot shorter? :)

File: plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/extensions/IrBodyFiller.kt [hunk 2]
```diff
@@ -218,6 +220,21 @@ private class DataFrameFileLowering(val context: IrPluginContext) : FileLowering
...
+
+    private fun computeNestedName(declaration: IrDeclarationWithName, result: StringBuilder): Boolean {
```
Comment (Jolanrensen) [inline plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/extensions/IrBodyFiller.kt:225]: How does this work when parents have backticks in the name? I suspect `` `Parent name`_`child name` `` won't compile, right?

**Review by Jolanrensen — APPROVED** [review]

File: plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/extensions/IrBodyFiller.kt (hunk 2 above)
Comment (koperagen) [reply plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/extensions/IrBodyFiller.kt:225]: Not sure yet. There are a few places where backtick identifiers can break things. I plan to review it later

Comment (Jolanrensen) [reply plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/extensions/IrBodyFiller.kt:225]: alright, don't forget it ;) haha

**Pull request merged**